packages/app-store/googlecalendar/api/add.ts

```diff
@@ -1,7 +1,7 @@
 import { google } from "googleapis";
 import type { NextApiRequest, NextApiResponse } from "next";
 
-import { WEBAPP_URL } from "@calcom/lib/constants";
+import { WEBAPP_URL_FOR_OAUTH } from "@calcom/lib/constants";
```

Comment:
We can do the same change in other integrations as well but we would need to test them first before making the change. I have tested Google Calendar, so adding it here.